modules/social_features/social_profile/modules/social_profile_privacy/social_profile_privacy.module

```diff
@@ -119,6 +139,11 @@ function social_profile_privacy_admin_settings_form_submit($form, FormStateInter
   }
 
   $config->save();
+
+  // We need to update profile names.
+  /** @var \Drupal\social_profile_privacy\Service\SocialProfilePrivacyBatchHelper $batch_helper */
+  $batch_helper = \Drupal::service('social_profile_privacy.batch.helper');
+  $batch_helper::bulkUpdateProfileNames();
```

Comment:
It can be called like this: `SocialProfilePrivacyBatchHelper::bulkUpdateProfileNames()`

Reply:
Updated.